admin/python/lsst/qserv/admin/configure.py

```diff
@@ -212,42 +212,43 @@ def _get_template_params():
-        'XROOTD_PORT': config['xrootd']['xrootd_port'],
-        'XROOTD_ADMIN_DIR': os.path.join(config['qserv']['qserv_run_dir'], 'tmp'),
-        'CMSD_MANAGER_PORT': config['xrootd']['cmsd_manager_port'],
-        'HOME': os.path.expanduser("~"),
-        'NODE_TYPE': config['qserv']['node_type'],
-        'WMGR_PORT': config['wmgr']['port'],
-        'WMGR_USER_NAME': random_string(string.ascii_letters + string.digits, 12),
-        'WMGR_PASSWORD': random_string(string.ascii_letters + string.digits + string.punctuation, 23),
-        'WMGR_SECRET_KEY': random_string(string.ascii_letters + string.digits, 57),
+            'CMSD_MANAGER_PORT': config['xrootd']['cmsd_manager_port'],
+            'COMMENT_MONO_NODE' : comment_mono_node,
+            'HOME': os.path.expanduser("~"),
+            'LD_LIBRARY_PATH': os.environ.get('LD_LIBRARY_PATH'),
+            'LUA_DIR': os.path.join(config['lua']['base_dir']),
+            'MYSQLD_DATA_DIR': os.path.join(config['qserv']['qserv_data_dir'], "mysql"),
+            # used for mysql-proxy in mono-node
+            'MYSQLD_HOST': '127.0.0.1',
+            'MYSQLD_PASSWORD_MONITOR': config['mysqld']['password_monitor'],
+            'MYSQLD_PASSWORD_ROOT': config['mysqld']['password_root'],
+            'MYSQLD_PORT': config['mysqld']['port'],
+            'MYSQLD_SOCK': config['mysqld']['sock'],
+            'MYSQLD_USER_MONITOR': config['mysqld']['user_monitor'],
+            'MYSQLD_USER_QSERV': config['mysqld']['user_qserv'],
+            'MYSQL_DIR': config['mysqld']['base_dir'],
+            'MYSQL_PROXY_PORT': config['mysql_proxy']['port'],
+            'NODE_TYPE': config['qserv']['node_type'],
+            'PATH': os.environ.get('PATH'),
+            'PYTHONPATH': os.environ['PYTHONPATH'],
+            'PYTHON_BIN': python_bin,
+            'QSERV_DATA_DIR': config['qserv']['qserv_data_dir'],
+            'QSERV_DIR': config['qserv']['base_dir'],
+            'QSERV_LOG_DIR': config['qserv']['log_dir'],
+            'QSERV_MASTER': config['qserv']['master'],
+            'QSERV_META_CONFIG_FILE': config['qserv']['meta_config_file'],
+            'QSERV_PID_DIR': os.path.join(config['qserv']['qserv_run_dir'], "var", "run"),
+            'QSERV_RUN_DIR': config['qserv']['qserv_run_dir'],
+            'QSERV_UNIX_USER': getpass.getuser(),
+            'SCISQL_DIR': scisql_dir,
+            'WMGR_PASSWORD': random_string(string.ascii_letters + string.digits + string.punctuation, 23),
+            'WMGR_PORT': config['wmgr']['port'],
+            'WMGR_SECRET_KEY': random_string(string.ascii_letters + string.digits, 57),
+            'WMGR_USER_NAME': random_string(string.ascii_letters + string.digits, 12),
+            'XROOTD_ADMIN_DIR': os.path.join(config['qserv']['qserv_run_dir'], 'tmp'),
+            'XROOTD_DIR': config['xrootd']['base_dir'],
+            'XROOTD_MANAGER_HOST': config['qserv']['master'],
+            'XROOTD_PORT': config['xrootd']['xrootd_port'],
```

Comment:
Better not to mix whitespace change with content change, it's very hard to see what has changed here.